### DISCUSSION TITLE
Open/Close all cells

### DISCUSSION THREAD

**Discussion by behinger** [fonsp/Pluto.jl#2547] [Questions about using Pluto] [answered]
Hey!
I was wondering if there is any functionality to (temporarily) open all hidden cells?

Sometimes I have a larger notebook for teaching with many closed cells - if I then need to search for one of these cells I have to manually click all the hidden cells and open them until I find it.

Thus a functionality like "temporarily open all cells" (+the button changes to "back to last status) in the statusbar would be super helpful!

Maybe this is already implemented, I guess it is actually not that difficult to do and I could even try at some point

Cheers, Bene

**Comment by unnamedunknownusername** [comment, marked as answer]
If I understand you correctly. You can already do this.

you can either left click and drag and select all cells 
![image](https://github.com/fonsp/Pluto.jl/assets/109242984/5a4b84e8-e00c-4440-a2e9-88fa3ab634be)

or click/unfocus/don't have any cell selected and use `ctrl+a` to select all cells
![image](https://github.com/fonsp/Pluto.jl/assets/109242984/d2b899a6-7ed8-4a83-8fd8-041638f1abae)

and then use use any one of the show/hide "eye buttons" to collapse/collapse/hide/unhide all the selected cells
e.g. after click in first cells eye
![image](https://github.com/fonsp/Pluto.jl/assets/109242984/8f7a81a7-d808-4024-958a-f995d8259d35)

hope that helps

**Reply by behinger** [reply]
oh wow. Supernice!

Only thing missing is to "save" a status and recover it, e.g. if half of your cells are open otherhalf closed, that you can open all of them, but then close only the ones that were closed before. But I feel like I can already work quite well with this solution
